test/helpers/setup-helper.js

```diff
@@ -12,6 +12,10 @@ global.chrome = {
 
 nock.disableNetConnect();
 nock.enableNetConnect('localhost');
+/* eslint-disable-next-line jest/require-top-level-describe */
+beforeEach(() => {
```

Comment:
This doesn't work in Jest because `beforeEach` isn't defined until after the test environment is created, which happens in the `setupAfterEnv` hook and not the `setup` hook. So this has to *also* go in `test/jest/setup.js`, otherwise we get a `beforeEach is not defined` error while running Jest in watch mode.

Reply:
Yeah that's what I was thinking with this comment: https://github.com/MetaMask/metamask-extension/pull/15717#discussion_r977801495

Reply:
Oops, I must have not seen this comment. Sounds good.